src/main/java/org/jabref/gui/DialogService.java

```diff
@@ -185,6 +186,13 @@ boolean showConfirmationDialogWithOptOutAndWait(String title, String content,
      */
     void showCustomDialog(BaseDialog<?> dialog);
 
+    /**
+     * Shows a custom window.
+     *
+     * @param window window to show
+     */
```

Comment:
This could be AI-generated :). No additional information.

Either add some text on the "why" (this is necessary) or remove it (removal is really OK)

Reply:
Ahah, no, it's symmetry.

I just saw you wrote comments for every method. What I did is I copied text from custom dialog, and replaced dialog with window)